receiver/varnishreceiver/internal/metadata/generated_metrics.go

```diff
@@ -32,7 +27,7 @@ func (ms *MetricSettings) Unmarshal(parser *confmap.Conf) error {
 	if err != nil {
 		return err
 	}
-	ms.enabledProvidedByUser = parser.IsSet("enabled")
+	ms.enabledSetByUser = parser.IsSet("enabled")
```

Comment:
Why is this name `enabledProvidedByUser` update ?

Reply:
It was changed in the generation templates: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/4524ffcac423dcd6743f012e3b24808e6346a7cd/cmd/mdatagen/templates/metrics.go.tmpl#L26
I don't really know why, I didn't pay much mind to it because it doesn't end up being used outside this generated package

Reply:
Thanks!